backend/config/corpora-api.yml

```diff
@@ -630,6 +667,11 @@ components:
           type: string
         detail:
           type: string
+    gene_sets:
+      type: array
+      items:
+        type: array
+        items: { }
```

Comment:
its variable. It will be 2 strings and a variable number of arrays
["geneset_name", "geneset_description", ["gene1", "gene1description (optional)", {"additionalParams":"alsoOptional"}], ["gene2]]